habitat_baselines/evaluate_ppo.py

```diff
@@ -5,20 +5,38 @@
 # LICENSE file in the root directory of this source tree.
 
 import argparse
+import os
+import time
 
+import cv2
+import numpy as np
 import torch
+from torch.utils import tensorboard
 
 import habitat
 from config.default import get_config as cfg_baseline
+from habitat import logger
 from habitat.config.default import get_config
+from habitat.utils.visualizations.utils import images_to_video
 from rl.ppo import PPO, Policy
-from rl.ppo.utils import batch_obs
+from rl.ppo.utils import batch_obs, frames_to_tb_video, generate_frame
 from train_ppo import make_env_fn
 
 
+def poll_checkpoint_folder(checkpoint_folder, previous_ckpt_ind):
+    assert os.path.isdir(checkpoint_folder), "invalid checkpoint folder path"
+    models = os.listdir(checkpoint_folder)
```

Comment:
I think `os.listdir` returns the filenames in a specific order, we might not needed to sort and check them. Also it would be useful to check if model has been successfully evaluated in a better manner, it could be the case that `ckpt23` will appear after `ckpt123` even though `ckpt123` is written after `ckpt23` and you would miss out on this detail. My suggestion is to order model files in the order of time they were created.

I see that below you are sorting them according to the index, it would be better to use timing I think instead of index, also can you add a comment to make this explicit.

Reply:
Just changed the sorting mechanism to be based on last modification time. Now user's checkpoint naming convention won't affect sorting, however this does require user to ckpts in a new dir for each new run or delete ckpts of old run.